### PR TITLE
Introduce Electron tests Example

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: .github/workflows/electron-tests.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        java-version: 17
+        cache: 'gradle'
+
+    - name: Run Electron tests
+      run: |
+        ./gradlew runStories -Pvividus.configuration.suites=electron \
+                             -Pvividus.configuration.profiles=desktop/electron,web/desktop/chrome
+
+    - name: Publish Electron tests report
+      if: always()
```
Comment (uarlouski) [inline .github/workflows/electron-tests.yml:42]: is this statement required?

Comment (ikalinin1) [reply .github/workflows/electron-tests.yml:42]: sure thing

**Comment by ikalinin1** [comment]
> Please update README as well

done

File: .github/workflows/electron-tests.yml (hunk 1 above)
Comment (uarlouski) [reply .github/workflows/electron-tests.yml:42]: okay, just forgot that steps are fail fast

**Review by uarlouski — APPROVED** [review]

**Review by valfirst — APPROVED** [review]

**Pull request merged**